tests/lib/clients/01-asyncio.py

```diff
@@ -91,6 +87,7 @@ async def main(self):
         await self.disconnected
         rc = 0
 
+
 rc = 1
 loop = asyncio.get_event_loop()
 loop.run_until_complete(AsyncMqttExample(loop).main())
```

Comment:
(i can't comment on right line). I think `exit(rc)` should be replaced by `assert rc == 0`. Without that pytest will succeed even when test fail.

Reply:
Well actually it's not enough. I'm trying by changing  line 88 to `rc = 1` (which make the test failing). pytest still works without error.

Reply:
By move those line inside a function named test is make test being run:
```
def test():
    loop = asyncio.get_event_loop()
    loop.run_until_complete(AsyncMqttExample(loop).main())
    loop.close()
    assert rc == 0
```

But we are missing to run the fake server which cause a timeout error

Reply:
After a second read, I think we just need to add `assert proc.wait() == 0` in test_01_asyncio.py